packages/core/src/components/Table/Table/__stories__/Table.stories.helpers.tsx

```diff
@@ -325,7 +325,7 @@ export const virtualizedScrollTableColumns: ITableColumn[] = [
   }
 ];
 
-export function sort(columnId: string, sortState: "asc" | "desc" | "none", tableData: []) {
+export function sort<T>(columnId: keyof T, sortState: "asc" | "desc" | "none", tableData: T[]) {
```

Comment:
how about adding some default? Wouldn't it break when no type is provided (all of the existing code using this method)

Reply:
It's needed to provide better type-safety for the `columnId` prop. It will not allow you to pass just any random string, only real keys of `T` is allowed.

It is not necessary to add default value for the `T`, since it will be automatically infered from the provided `tableData` prop

Reply:
@talkor do you still think default value necessary in this case? 1) It's an internal storybook-related file, it is not exposed from the library 2) type if determined automatically from the specified props in this case